test/unit/simplestyle.js

```diff
@@ -5,21 +5,19 @@ const properties = {
   'fill-opacity': 0.5
 };
 
+let style = simplestyleToLeafletStyle(properties, 'Polygon');
```

Comment:
I fixed the simplestyle unit test and then realized that we could call the mapping method globally because properties will be changed during the three method calls and the third test was not making much sense like that, i.e. having already all the default values within properties object: `{"weight":0,"fillColor":"#c31906","fillOpacity":0.35}`. Do you agree with that change?